Dockerfile.base

```diff
@@ -11,7 +11,6 @@ RUN set -eux && \
     rustup component add rust-src --toolchain nightly-2022-04-20-x86_64-unknown-linux-musl && \
     rustup component add rustfmt --toolchain nightly-2022-04-20-x86_64-unknown-linux-musl && \
     rustup component add rust-std && \
-    # rustup target add wasm32-unknown-unknown && \
```

Comment:
Made a change just for publishing `iroha2-base:dev` on merge, because:

- `on.workflow_dispatch` is not available because `iroha2-dev` branch is not default

  > To trigger the workflow_dispatch event, your workflow must be in the default branch.
  > https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

- The experimental `on.pull_request_target` did not work for some reason

Reply:
It's amazing how broken GitHub actions are compared to e.g. GitLab CI, still, if it works, it works.